src/main/scanLogic/scanRunners/applicabilityScan.ts

```diff
@@ -90,20 +91,23 @@ export class ApplicabilityRunner extends BinaryRunner {
         if (issues) {
             // Generate applicable data for all the issues
             issues.forEach(analyzeIssue => {
```

Comment:
Let's add the types whenever we can
```suggestion
            issues.forEach((analyzeIssue: AnalyzeIssue) => {
```